app/models/event.py

```diff
@@ -61,7 +61,7 @@ class Event(db.Model):
     large_image_url = db.Column(db.String)
     icon_image_url = db.Column(db.String)
     organizer_name = db.Column(db.String)
-    is_map_shown = db.Column(db.Integer)
```

Comment:
@shubham-padia 
It was already done in [PR#3820](https://github.com/fossasia/open-event-orga-server/pull/3820/files) , just renaming was missed. 
Yes , it was changed from **Integer** to **Boolean** only it was forgotten to modify here.

Check [this migration here](https://github.com/fossasia/open-event-orga-server/pull/3820/files#diff-36a2b03dc3c9199fef623a0a29d63442R41)

![image](https://user-images.githubusercontent.com/17861054/27467937-2de4a8fa-5804-11e7-95a0-eb81f40accb7.png)